framework_tvb/tvb/core/services/import_service.py

```diff
@@ -91,6 +91,10 @@ def is_old_form(self):
     def order_field(self):
         return self.operation.create_date if (self.operation is not None) else datetime.now()
 
+    @property
+    def order_field_by_folder(self):
+        return int(os.path.basename(self.operation_folder)) if (self.operation_folder is not None) else datetime.now()
+
```

Comment:
This conversion to int is not reliable. We might have links, for example.
Can't we continue to order by operation create date?

Reply:
I think I created some faulty zip data, but after further testing the initial solution is the wanted one.